llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -78,6 +78,11 @@ static cl::opt<LoopVectorizeHints::ScalableForceKind>
                 "Scalable vectorization is available and favored when the "
                 "cost is inconclusive.")));
 
+static cl::opt<bool> UseWiderVFIfVariantsPresent(
```

Comment:
I feel like this option should really live in LoopVectorize.cpp since it's a cost model issue rather than a legality issue.